telegram/_bot.py

```diff
@@ -2620,7 +2625,8 @@ async def answer_inline_query(
             :paramref:`telegram.InlineQuery.answer.auto_pagination` set to :obj:`True`, which will
             take care of passing the correct value.
 
-        .. seealso:: :attr:`telegram.InlineQuery.answer`
+        .. seealso:: :attr:`telegram.InlineQuery.answer`, `Your First Bot <https://github.com/\
+            python-telegram-bot/python-telegram-bot/wiki/Extensions-–-Your-first-Bot>`_
```

Comment:
The tutorial is so generic IMO that it suffices to link to it from the very fundamental classes in the library. Maybe only `Bot`, `Application`, `ApplicationBuilder`, `Update`

Reply:
Done

Reply:
this should be moved from `answer_inline_query` to the actual class doc

Reply:
Done

Reply:
I will also move links to "Your First Bot" and "Architecture Overview" from `Application.add_handler()` to class docstring of `Application`